homeassistant/components/switch/pilight.py

```diff
@@ -0,0 +1,216 @@
+"""
+
+Support for Pilight Switches.
+"""
+import logging
+import socket
+import json
+
+from homeassistant.components.switch import SwitchDevice
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = []
+
+
+def setup_platform(hass, config, add_devices_callback, discovery_info=None):
+	""" Sets up a pilight socket client """
+	#_LOGGER.info(config)
+	host = config['ip']
+	port = config['port']
+	server = Pilight_Server(host, port)
```

Comment:
Here, a feedback would be nice if the configuration variables are wrong or the server is offline.